followthemoney/types/language.py

```diff
@@ -85,7 +91,7 @@ class LanguageType(EnumType):
     LANGUAGES = get_env_list("FTM_LANGUAGES", LANGUAGES)
```

Comment:
This is marked as constant coz it's all uppercase, but we're overwriting it on the line below. Is this legit?

Reply:
`constant-ish`. It's sort of stable at run-time, is what I mean.